docs/components/query.md

```diff
@@ -8,7 +8,7 @@ menu: components
 
 The `thanos query` command (also known as "Querier") implements the [Prometheus HTTP v1 API](https://prometheus.io/docs/prometheus/latest/querying/api/) to query data in a Thanos cluster via PromQL.
 
-In short, it gathers the data needed to evaluate the query from underlying [StoreAPIs](../../pkg/store/storepb/rpc.proto), evaluates the query and returns the result.
+In short, it gathers the data needed to evaluate the query from underlying [StoreAPIs](/pkg/store/storepb/rpc.proto), evaluates the query and returns the result.
```

Comment:
I forgot to comment. 🤦🏽 

 We need to put absolute link otherwise it won't work on website. (E.g see CI status check `netlify preview` - it will show you how website will look 🤗

Reply:
Thank you for your comment 🤗

Isn't it absolute link `/pkg/store/storepb/rpc.proto` ? I checked CI status check details (netlify/thanos-io/deploy-preview) and it works as expected.

![Jan-12-2021 19-24-07](https://user-images.githubusercontent.com/16697306/104302158-cd29c500-550b-11eb-9d54-de4c36e1ef44.gif)

Can you check once more about this change?